docs/sphinx/conf.py.in

```diff
@@ -40,7 +40,7 @@ from datetime import datetime
 extensions = ['sphinx.ext.extlinks', 'edit_on_github']
 
 ## Configuration for the edit_on_github extension
-edit_on_github_project = 'openmicroscopy/bioformats'
+edit_on_github_project = 'ome/files'
```

Comment:
@rleigh-dundee : I had the same thought re: the suffix. I think the last time we discussed we said that worst case we would join the git history some future `ome-files-java` to create a single `ome-files` if need be.